clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -653,6 +654,33 @@ public void produceResponseVersionTest() {
         assertEquals("Response data does not match", responseData, v2Response.responses());
     }
 
+    @Test
+    public void produceResponseRecordErrorsTest() {
```

Comment:
Maybe this should be in `ProduceResponseTest`? We started writing this type of test in dedicated classes as this class was getting too large.

Reply:
@ijuma Thanks for the review. I have moved this test and the other two produce response tests to `ProduceResponseTest`.